apps/css-workshop/pages/tree.html

```diff
@@ -1239,6 +1246,1300 @@ <h2>With Icons</h2>
       </ul>
     </section>
 
+    <hr />
+    <h2>Sizes</h2>
+    <section id="demo-sizes">
+      <ul
+        class="iui-tree"
+        role="tree"
+        data-iui-size="small"
+      >
+        <li
+          class="iui-tree-item"
+          role="treeitem"
+          tabindex="0"
+          aria-expanded="true"
+        >
+          <div class="iui-tree-node">
+            <input
+              class="iui-checkbox iui-checkbox-visibility iui-tree-node-checkbox"
+              type="checkbox"
+              checked
+            />
+            <div class="iui-tree-node-content">
+              <button
+                class="iui-button"
+                data-iui-variant="borderless"
+                data-iui-size="small"
+                type="button"
+              >
+                <svg-chevron-right-small
+                  class="iui-button-icon iui-tree-node-content-expander-icon  iui-tree-node-content-expander-icon-expanded"
+                  aria-hidden="true"
+                ></svg-chevron-right-small>
+              </button>
+              <svg-placeholder
+                class="iui-svg-icon iui-tree-node-content-icon"
+                data-iui-icon-size="s"
+                aria-hidden="true"
+              ></svg-placeholder>
+              <span class="iui-tree-node-content-label">
+                <div class="iui-tree-node-content-title">Facility 1</div>
+              </span>
+            </div>
+          </div>
+          <ul
+            class='iui-sub-tree'
+            role="group"
+          >
+            <li
+              class="iui-tree-item"
+              role="treeitem"
+              tabindex="0"
+              aria-expanded="true"
+            >
+              <div
+                class="iui-tree-node"
+                style='--level: 1;'
```

Comment:
am i understanding correctly that the old markup still works for existing features?

the preferable scenario would be:
1. `--level` on tree-node continues to work for tree without lines
2. `--level` on sub-tree works for all cases, with and without lines